src/Language/Docker/Parser/Copy.hs

```diff
@@ -78,6 +98,12 @@ chown = do
   ch <- someUnless "the user and group for chown" (== ' ')
   return $ Chown ch
 
+chmod :: Parser Chmod
+chmod = do
+  void $ string "--chmod="
+  ch <- someUnless "the mode for chmod" (== ' ')
+  return $ Chmod ch
+
```

Comment:
Very naive question, I don't usualy read Haskell, even though this is scoped, would it be worth changing 'ch' var to 'chm' and 'cho' respectively, to match changes above, to avoid confusion ?

Reply:
IMO no, not in and of itself. The `ch`s are local to the scope of the function. But since I had to make a fixup commit anyways, I changed it to avoid the confusion.